### PR TITLE
Add add client

### PR DESCRIPTION
**Pull request by dangron**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -2,14 +2,20 @@
     "name": "space48/rialtowebservice",
     "description": "Rialto Web Service Soap Package",
     "require": {
-        "php": ">=5.3.3",
+        "php": ">7.0",
```
Comment (joshdifabio) [inline composer.json]: Should probably be >=7.0?

Comment (dangron) [reply composer.json]: yes!

File: src/Client/AddClient.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    /** @var string */
+    private $username;
+
+    /** @var string */
+    private $password;
+
+    /** @var string */
+    private $wsdlUrl;
+
+    public function __construct(string $username, string $password, bool $testMode = false)
```
Comment (joshdifabio) [inline src/Client/AddClient.php]: Optional params are smelly. Better to inject the URL instead of an optional flag.

Comment (dangron) [reply src/Client/AddClient.php]: ok!

**Pull request merged**